.vite/deps/_metadata.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Please add `.vite` to your `.gitignore` and execute `git rm .vite` once to remove it from your git.